lib/index.d.ts

```diff
@@ -1,4 +1,3 @@
-/// <reference types="node" />
```

Comment:
I do not know what causes this change or whether it is significant. (Not sufficiently familiar with TypeScript.)

Reply:
Probably a typescript compiler version change due to you using npm rather than yarn, and thus not picking up the `yarn.lock` version. Not a major concern.

Reply:
Indeed, it has returned with 3.7.3 that is used now.